src/data_structures/message.cpp

```diff
@@ -7,19 +7,19 @@
 #include "../../include/data_structures/message.hpp"
 
 /***************************************************/
-/************* Output stream ************************/
+/************* Output stream ***********************/
 /***************************************************/
 
-ostream& operator<<(ostream& os, const Message_t& msg) {
-  os << msg.value;
-  return os;
+ostream& operator<<(ostream& os, const message_t& msg) {
+    os << msg.value;

```

Comment:
name of the variable "os" could be "out_stream" to be more descriptive.

Reply:
Many thanks for this feedback. It is resolved now.